<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsWindow</class>
 <widget class="QDialog" name="SerialSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Communication Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="portlabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="portBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="baudrateLabel">
       <property name="text">
        <string>BaudRate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="databitsLabel">
       <property name="text">
        <string>DataBits:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="dataBitsBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="parityLabel">
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="parityBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stopbitsLabel">
       <property name="text">
        <string>StopBits:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="stopBitsBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="querymodeLabel">
       <property name="text">
        <string>QueryMode:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="queryModeBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="timeoutLabel">
       <property name="text">
        <string>Timeout:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="timeoutBox">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
